setup.py

```diff
@@ -21,7 +21,7 @@
         "write_to": "python/lsst/ts/dsm/version.py",
         "write_to_template": scm_version_template,
     },
-    setup_requires=["setuptools_scm", "pytest-runner"],
+    setup_requires=["setuptools_scm"],
```

Comment:
Will setup.py still run unit tests? If not, do we care? (I suspect not, as long as Jenkins runs the tests).

Also when I read the pytest-runner deprecation notice it suggested not specifying the `test_requires` argument to `setuptools.setup`. Should you do that here?

I'd like to get rid of pytest-runner in all of my packages, as well and I'm just trying to learn how we've decided to do it.

Reply:
We found that removing `pytest-runner` had no adverse effect on running the unit tests when building the conda packages, so it should be safe to remove. That being said, the package we tested this on was pretty simple as is the DSM so YMMV.

Reply:
I asked @couger01 and he said it's fine. I'll start doing that to my packages as I work on them.